src/test/java/it/pagopa/pn/bff/pnclient/auth/PnAuthFleetClientImplTest.java

```diff
@@ -0,0 +1,95 @@
+package it.pagopa.pn.bff.pnclient.auth;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.pagopa.pn.bff.mocks.AuthFleetMock;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.mockserver.client.MockServerClient;
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.model.MediaType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import reactor.test.StepVerifier;
+
+import static org.mockserver.integration.ClientAndServer.startClientAndServer;
+import static org.mockserver.model.HttpRequest.request;
+import static org.mockserver.model.HttpResponse.response;
+
+@SpringBootTest(properties = {
```

Comment:
replace with

```
@SpringBootTest
@TestPropertySource(locations = "classpath:application-test.properties")
```

and put the needed configuration into the application-test.properties file.
maybe the `cors.allowed.domains=http://localhost:8080` is not necessary